parameter.go

```diff
@@ -0,0 +1,230 @@
+package tfe
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/url"
+)
+
+// Compile-time proof of interface implementation.
+var _ Parameters = (*parameters)(nil)
+
+// Parameters describes all the parameter related methods that the Terraform
+// Enterprise API supports.
+//
+// TFE API docs: https://www.terraform.io/docs/enterprise/api/policy-set-params.html
+type Parameters interface {
+	// List all the parameters associated with the given policy-set.
+	List(ctx context.Context, policySetID string, options ParameterListOptions) (*ParameterList, error)
+
+	// Create is used to create a new parameter.
+	Create(ctx context.Context, policySetID string, options ParameterCreateOptions) (*Parameter, error)
+
+	// Read a parameter by its ID.
+	Read(ctx context.Context, policySetID string, parameterID string) (*Parameter, error)
+
+	// Update values of an existing parameter.
+	Update(ctx context.Context, policySetID string, parameterID string, options ParameterUpdateOptions) (*Parameter, error)
+
+	// Delete a parameter by its ID.
+	Delete(ctx context.Context, policySetID string, parameterID string) error
+}
+
+// parameters implements Parameters.
+type parameters struct {
+	client *Client
+}
+
+// ParameterList represents a list of parameters.
+type ParameterList struct {
+	*Pagination
+	Items []*Parameter
+}
+
+// Parameter represents a Terraform Enterprise parameter.
+type Parameter struct {
+	ID        string       `jsonapi:"primary,vars"`
+	Key       string       `jsonapi:"attr,key"`
+	Value     string       `jsonapi:"attr,value"`
+	Category  CategoryType `jsonapi:"attr,category"`
+	Sensitive bool         `jsonapi:"attr,sensitive"`
+
+	// Relations
+	PolicySet *PolicySet `jsonapi:"relation,configurable"`
+}
+
+// ParameterListOptions represents the options for listing parameters.
+type ParameterListOptions struct {
```

Comment:
I know it's long, but for clarity I would call this `PolicySetParameterListOptions`